resources/analysisTools/snvPipeline/convertToStdVCF.py

```diff
@@ -0,0 +1,490 @@
+    #"VALIDATED": {"number":-1,  "type": "?",        "description": "Validated by follow-up experiment"},
+    #"1000G":     {"number": 0,  "type": "Flag",     "description": "Membership in 1000 Genomes"},
+
+    #-----------------------------------------------------------
+    # DKFZ in-house INFO fields
+    # partially already in INFO, partially present as INFO_control, partially as freestanding column.
+    #
+    # Freestanding columns must have a "new_info_id" to be extracted into the new spec-conforming VCF's INFO field.
+    # Their key must be the column header name as it occurs in the in-house input file.
+    #-----------------------------------------------------------
+    "DP_ctrl":               {"number": 1,  "type": "Integer",
+                              "description": "Total depth of control"},
+    "DP4":                   {"number": 4,  "type": "Integer",
+                              "description": "Counts of REF and ALT reads on forward and reverse strand"},
+    "DP5_ctrl":              {"number": 5,  "type": "Integer",
+                              "description": "Counts of ?"},
+    "MQ_ctrl":               {"number": 1,  "type": "Float",
+                              "description": "Mapping quality of Control"},
+    "ACGTNacgtnPLUS":        {"number": 10, "type": "Integer",
+                              "description": "Nucleotide counts on Plus Strand"},
+    "ACGTNacgtnMINUS":       {"number": 10, "type": "Integer",
+                              "description": "Nucleotide counts on Minus Strand"},
+    "ACGTNacgtnHQ_ctrl":     {"number": 10, "type": "Integer",
+                              "description": "Nucleotide counts on ?"},
+    "ACGTNacgtn_ctrl":       {"number": 10, "type": "Integer",
+                              "description": "Nucleotide counts on ?"},
+    "GENE":                  {"number": 1,  "type": "String",
+                              "description": "GeneSymbol if variation overlaps gene",
+                              "new_info_id": "GENE"},
+    "EXONIC_CLASSIFICATION": {"number": 1,  "type": "String",
+                              "description": "Functional implication if variation is located with respect to exon",
+                              "new_info_id": "EXONICCL" },
+    "ANNOVAR_TRANSCRIPTS":   {"number": 1,  "type": "String",
+                              "description": "Details of non-synonymous variation's impact on protein",
+                              "new_info_id": "ANNOVARTR"},
+    "CONFIDENCE":            {"number": 1,  "type": "Integer",
+                              "description": "Empirical confidence level of call, ten levels (1-10), >7=trustworthy",
+                              "new_info_id": "CONF"},
+    "RECLASSIFICATION":      {"number": 1,  "type": "String",
+                              "description": "Reclassification of the germline/somatic state based on additional information",
+                              "new_info_id": "RECL"},
```

Comment:
The script runs through. I would like to have the same column names in the INFO column as well. 
Not like `RECLASSIFICATION` -> `RECL`
It will be helpful for the users who are already familiar with the column names.

Reply:
I understand. Basically, I just moved Jules' and Sophias' script into the repository, but I guess these fields are freely named, right? I'll have to have a look at the specification.

One has to be aware that the keywords are repeated every line. After compression, though, the difference may be negligible.

Reply:
Actually, in the original code this data structure was in a Python 
separate file. This is just nested dicts. Maybe I can load it from a JSON or YAML, which would make the output configurable.

Reply:
That would be better. We may use this script to reformat the output files to proper VCF formats in future versions.

Reply:
I extracted the static configuration into a JSON file. The default JSON file has the same configuration as before and is expected to reside besides the Python script.

An alternative config can be used as optional 4 argument to the program call.

The fields have not yet been updated. I'm still not sure whether this should be done at all. I will try with same bigger VCFs and see how much the compression can compensate for the repetition of lengthy names.